### PR TITLE
[AR-8207] impl logic to switch to global without verification

### PR DESCRIPTION
**Pull request by karthik-durai**
# Pull Request

Resolves or continues [AR-8207](https://team-1624093970686.atlassian.net/browse/AR-8207).

## Changes

1. Remove verification step to switch to Global keyspace.

## Checklist

- [X] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [X] The changes have been tested locally.

[AR-8207]: https://team-1624093970686.atlassian.net/browse/AR-8207?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/app-configure/AppKeyspace.vue [hunk 1]
```diff
@@ -141,33 +137,6 @@ function handleCancel() {
                     >
                       *Recommended
```
Comment (shrinathprabhu) [inline src/components/app-configure/AppKeyspace.vue:138]: Remove this block, we don't recommend it anymore

File: src/components/app-configure/AppKeyspace.vue [hunk 2]
```diff
@@ -111,9 +111,6 @@ function handleCancel() {
         <div v-if="chosenWalletUIMode !== CUSTOM_UI" class="relative">
           <div
             class="keyspace-card"
-            :class="{
-              'disabled-card gradient-border-card': app.status !== 2,
-            }"
             @click.stop="
               app.status === 2 ? (selectedKeyspace = 'global') : void 0
```
Comment (shrinathprabhu) [inline src/components/app-configure/AppKeyspace.vue:115]: Need to update this logic as well

Comment (shrinathprabhu) [reply src/components/app-configure/AppKeyspace.vue:115]: click.stop="selectedKeyspace='global'"

Comment (karthik-durai) [reply src/components/app-configure/AppKeyspace.vue]: [done](https://github.com/arcana-network/developer-dashboard/pull/431/commits/2cfcff59411aab2d20a21eed12b045baf10e250e)

File: src/components/app-configure/AppKeyspace.vue (hunk 1 above)
Comment (karthik-durai) [reply src/components/app-configure/AppKeyspace.vue]: [done](https://github.com/arcana-network/developer-dashboard/pull/431/commits/2cfcff59411aab2d20a21eed12b045baf10e250e)

**Review by shrinathprabhu — APPROVED** [review]

**Pull request merged**